myapp/spec/system/tasks_system_spec.rb

```diff
@@ -3,27 +3,34 @@
 require 'rails_helper'
 
 RSpec.describe 'Tasks (System)', type: :system do
-  it 'tasks is ordered by created_at with descending order' do
-    expected_order = %w[task2 task1]
+  let!(:task1) { create(:task, name: 'task1', status: 2, priority: 0, label:'aaa', due_date: Faker::Time.forward(days: 1, period: :evening) ) }
+  let!(:task2) { create(:task, name: 'task2', status: 2, priority: 0, label:'aaa', due_date: Faker::Time.forward(days: 2, period: :evening) ) }
```

Comment:
I got random error when running rspec and I done some investigation:

It seemed that the `days` parameter is the maximum days to go in future. So even you set days: 2 on the second task, in some case it will just go forward 0 days:

![image](https://user-images.githubusercontent.com/10822260/123191375-2266fb00-d4dc-11eb-84ef-2f384ce4b279.png)

https://www.rubydoc.info/github/stympy/faker/Faker/Time#forward-class_method

If you doesn't need random time for comparison, maybe can stick with basic Rails' Time class:
`Time.zone.now + 1.day`

Reply:
i was wondering the same thing why it sometimes passes but other time fails